inginious/frontend/plugins/register_students/static/email_template.html

```diff
@@ -28,7 +28,11 @@ <h2>$:_("Welcome on UNCode")</h2>
             <li>$:_("To add the Google Authentication method, select 'Google' then click on 'Add new binding' button.")</li>
             <li>$:_("After adding Google Authentication you can log in to UNCode using either the normal Authentication (username and password) or Google Authentication.")</li>
         </ol>
+        <br><br>
+        $:_("Don't forget to check UNCode")
```

Comment:
```suggestion
        $:_("See the UNCode's ")
```
Probably this is better?